contracts/Hub.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.7;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Hub Contract that coordinates Storage, Fund and Minter contract.
+ * @dev The contract can mint new synthtic assets (erc20 token), check the collateral ratio of addresses
+ * and liquidate addresses if the collateral ratio falls below 150%
+ * @author The Everest team.
+ */
+
+contract Hub is Ownable {
+    // Contract addresses
+    ICollateralFunds private fundContract;
```

Comment:
I think we should swith all the variables from `private` to `public`.

Reply:
What's the advantage?

Reply:
We'll have getters on every variables, that's more transparent for us and users.